### PR TITLE
Localstorage blocked warning screen

### PR DESCRIPTION
**Pull request by Marik-D**
# Summary
A description of the changes proposed in the pull request.

Fixes #number

## Checklist
- [ ] Change is small and easy to review (please split big changes into multiple PRs)
- [ ] Change is consistent with architecture
- [ ] Change is consistent with test architecture
- [ ] Change is consistent with naming conventions
- [ ] New code is covered with tests
- [ ] Tests related to old code are updated
- [ ] Documentation is up to date with changes

### CODE REVIEW

**Comment by JustynaBroniszewska** [comment]
We should update the tests. Let's do it tomorrow.

File: universal-login-react/src/app/BrowserChecker.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export class BrowserChecker {
+  isLocalStorageBlocked() {
+    try {
+      if (!('localStorage' in window)) {
+        return true;
+      }
+      if (window.localStorage === undefined) {
+        return true;
+      }
```
Comment (xitronix) [inline universal-login-react/src/app/BrowserChecker.ts]: Why no 
if(!!window.localStorage) {
  return true;
}

File: universal-login-wallet/src/index.tsx [hunk 2]
```diff
@@ -17,9 +17,7 @@ const AppBootstrapper = () => {
 
     services.walletService.loadFromStorage();
     setBetaNotice(services.sdk);
-
-    const isIncognito = await isPrivateMode();
-    return {services, isIncognito};
+    return {services};
```
Comment (xitronix) [inline universal-login-wallet/src/index.tsx]: we can now return services without object wrapping , so we would not have to extracting services after

File: universal-login-web3/src/ULWeb3Provider.ts [hunk 3]
```diff
@@ -52,6 +54,7 @@ export class ULWeb3Provider implements Provider {
     applicationInfo,
     uiInitializer = initUi,
     storageService = new StorageService(),
+    browserChecker = new BrowserChecker(),
```
Comment (xitronix) [inline universal-login-web3/src/ULWeb3Provider.ts]:
```suggestion
    private browserChecker = new BrowserChecker(),
```

**Pull request merged**